core/tests/test_core_middleware.py

```diff
@@ -4,11 +4,11 @@
 
 from django.http import HttpResponse
 
-from ui import middleware
```

Comment:
don't think we need `test_core_middleware.py` name, it's already under `core` folder

Reply:
pytest sometimes gets confused when it encounters test files with the same name and something like "tried to import blah but __file__ is not what we expected"